content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -1,121 +1,83 @@
 # Sharing Experiments
 
-There are two types of remotes that can store experiments. Git remotes are
-distributed copies of the Git repository, for example on GitHub or GitLab.
+If your team uses Git server or hosting (e.g. GitHub, GitLab, etc.) to
+collaborate on projects, you can also use them to save and share DVC
+Experiments.
```

Comment:
Experiments -> experiments?

Reply:
It's "DVC Experiments" (branded). Unsure whether this will stay in the long run but it's how we're differentiating regular exps from our implementation for now (we've discussed this I believe).